releasenotes/notes/rcm-fix-disable-f45787a4e6576f53.yaml

```diff
@@ -0,0 +1,5 @@
+---
+issues:
```

Comment:
is this worthy of a release note? The commit is listed as a `fix` and this is an `issues` 🤔